tests/rptest/tests/tombstone_removal_e2e_test.py

```diff
@@ -139,6 +139,7 @@ def get_first_offset(self, node: ClusterNode) -> int:
         self.redpanda.logger.debug("partition is empty")
         return -1
 
+    @skip_debug_mode
```

Comment:
can we have this test to write less data in debug mode instead of skipping it completely ?